shared/services/rocketpool/client.go

```diff
@@ -62,6 +64,13 @@ const (
 	DebugColor = color.FgYellow
 )
 
+// When printing sync percents, we should avoid printing 100%.
+// This function is only called if we're still syncing,
+// and the `%0.2f` token will round up if we're above 99.99%.
+func SyncRatioToPercent(in float64) float64 {
```

Comment:
might make sense to move this to a different util file, but for now i shoved it here